src/EnergyPlus/CurveManager.cc

```diff
@@ -2578,116 +2546,116 @@ namespace CurveManager {
         // Return value
         Real64 CurveValue;
```

Comment:
With inheritance, this would be a virtual function. Also why is this function called `PerformanceCurveObject`? Should it not be called `CurveValue`?

Reply:
Yep, that's the plan.  And the naming is such that `CurveValue(int index, ...)` is the "API" for the curve module.  Internally, it can split between a table interpolation or a curve evaluation.  The `PerformanceCurveObject` function as it is named now is really what will be split into overridden functions when a structure emerges.